JobConfig/mixing/filters/IPAMuminusMichel.fcl

```diff
@@ -0,0 +1,7 @@
+# remove steps descendant from IPA michels generated by G4
+physics.producers.MuStopPileupFilter.selection: {
+  tool_type: ProcessVolumeDetectorStepAntiSelectionTool
+  processes: [ "DIO" ]
+  volumes: [ "protonabs1" ]
+  momentum_threshold: 1.0
```

Comment:
Is this momentum threshold value intended to be overridden in a real mixing job? If so I suggest to leave it blank, so that downstream jobs will fail unless it is set to a (correct) value. If you need a value for testing, you can set that in the test script.

Reply:
I think that this is the right place to define it. But it is good that you flagged it, because this needs to be updated to units of energy (already merged into `Offline`) and to match the selection of the primaries generated.

Reply:
Do we always intend this to then be the same value? If not it would need to be overridden and setting this to nil to then be overridden in the gen_Mix might make sense.

Reply:
@sophiemiddleton these will be changed at exactly the frequency that the configuration of the primary dts jobs are changed. e.g., if were to generate DIO tail electrons with an 85 MeV threshold, then we'd need to configure the mixing job to filter out DIO pileup with an 85 MeV threshold.

Given that, I think we should follow a common pattern between the primary jobs and their corresponding mixing jobs. Because the former have their energy ranges ``hard-coded'' (in `Production/primary/{DIOtail,IPAMuminusMichel}.fcl`), I've hard-coded the same values here. Dave and I talked a bit about this earlier, and it doesn't seem like there is a great solution to keep them synchronized. My understanding is that the former will be overwritten by yet-higher-level logic when the campaign is actually run. If that is the case, then IMO we should override these in the same way, and the (almost) single point of maintenance is at the campaign level. What do you think?

Reply:
OK, sounds like its the best we can do for now